src/mission_start.cpp

```diff
@@ -1719,3 +1719,34 @@ void mission_start::reveal_cathedral( mission *miss )
 {
     reveal_any_target( miss, { "cathedral_1", "museum" } );
 }
+
+void mission_start::reveal_refugee_center( mission *miss )
```

Comment:
The function is pretty generic. It's guts could be extracted into a function that accepts `"evac_center_18"` (and possibly `"road"`) and has a less specific name. Even better: it could be unified with `reveal_target()`/`reveal_route()`, and also accept a "revealer".